projects/chaos/src/relay.c

```diff
@@ -0,0 +1,50 @@
+#include "relay.h"
+
+#include <stdbool.h>
+
+#include "can_msg_defs.h"
+#include "event_queue.h"
+#include "relay_fsm.h"
+#include "relay_id.h"
+
+static FSM s_solar_front_relay_fsm;
```

Comment:
Something about this makes me sad )':

I personally prefer arrays but that approach isn't much better either.

Reply:
Done. I switched to an array.